tests/ParserTest.php

```diff
@@ -1233,12 +1234,12 @@ public function lonelyImport(): void
 
     public function escapedSpecialCaseTokens(): void
     {
-        $document = $this->parsedStructureForFile('escaped-tokens');
+        $document = self::parsedStructureForFile('escaped-tokens');
         $contents = $document->getContents();
         $rules = $contents[0]->getRules();
         $urlRule = $rules[0];
         $calcRule = $rules[1];
-        self::assertTrue(\is_a($urlRule->getValue(), '\\Sabberworm\\CSS\\Value\\URL'));
-        self::assertTrue(\is_a($calcRule->getValue(), '\\Sabberworm\\CSS\\Value\\CalcFunction'));
+        self::assertInstanceOf(URL::class, $urlRule->getValue());
+        self::assertInstanceOf(CalcFunction::class, $calcRule->getValue());
```

Comment:
Maybe `assertInstanceOf` wasn't around at the time these tests were implemented.  Good improvement.